AnalyticsApp/AnalyticsApp/ContentView.swift

```diff
@@ -90,6 +90,16 @@ struct ContentView: View {
             CustomButton(title: "Read AnonymousId") {
                 print("Current Anonymous Id: \(String(describing: AnalyticsManager.shared.anonymousId))")
             }
+            
+            HStack {
+                CustomButton(title: "Reset") {
+                    AnalyticsManager.shared.reset(false)
```

Comment:
```suggestion
                    AnalyticsManager.shared.reset()
```
This is just to demonstrate that making a call without the boolean flag is also a valid code.

Reply:
This change is in the sample app. It will be modified in future.